<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileInfoDisplayWidget</class>
 <widget class="QWidget" name="FileInfoDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 设置 QLabels 的基本样式 */
QLabel {
    font-size: 14px;
    font-weight: bold;
    color: #333;
    padding: 5px 0;
}

/* 针对显示文件类型标签的 QLabel */
QLabel#file_type {
    font-size: 12px;
    color: #666;
    background-color: #F5F5F5;
    border-radius: 4px;
    padding: 5px;
}

/* 设置文件路径和文件名的显示 */
QLabel#filename_label, QLabel#file_locations, QLabel#filesize_label {
    color: #333;
    font-size: 13px;
    padding: 5px;
    background-color: #FFF;
    border: 1px solid #DCDCDC;
    border-radius: 4px;
}

/* 设置用于显示图标的 QLabel */
QLabel#icon_label {
    width: 40px;
    height: 40px;
    background-color: #E0E0E0;
    border-radius: 4px;
    border: 1px solid #C0C0C0;
}

QTextBrowser {
    font-size: 13px;
    color: #333;
    background-color: #FFF;
    border: 1px solid #DCDCDC;
    border-radius: 4px;
    padding: 10px;
    margin-top: 10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>184</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="icon_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>184</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="filename_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="file_locations">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="file_type">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="filesize_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="image_display">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
